tests/core/process.c

```diff
@@ -710,11 +718,43 @@ store_export_address_if_mach_msg (const GumExportDetails * details,
 
 #endif
 
+static GThread *
+new_sleeping_dummy_thread_sync (volatile gboolean * done)
+{
+  TestThreadSyncData sync_data;
+  GThread * thread;
+
+  g_cond_init (&sync_data.cond);
+  g_mutex_init (&sync_data.mutex);
```

Comment:
Fields should be initialized in the same order that they’re declared in, i.e. `mutex` and then `cond`